### DISCUSSION TITLE
Clase 26 Agosto - Viernes Noche - Grabación

### DISCUSSION THREAD

**Discussion by matiastito** [dds-jv/foro#381] [Consultas generales] [answered]
Buenos días, primero perdón por molestar.
Espero esten bien. Estoy cursando los viernes a la noche y ayer no pude terminar de ver la clase. Recién accedí al campus y ví que todavía no la publicaron, mas o menos, ¿cuándo tienen planificado subirla? 

Muchas gracias!!!!

**Comment by gastonprieto** [comment, marked as answer]
buenas

ya deberia estar subido
